Sekoia.io/action_get_an_alert.json

```
@@ -15,6 +15,12 @@
       "stix": {
         "in": "query",
         "type": "boolean"
+      },
+      "cases": {
+        "in": "query",
+        "type": "boolean",
+        "description": "Get the cases associated with this alert",
+        "default": true
```

I think that the philosophy of this boolean is to not return the cases associated to an alert by default.
```suggestion
        "default": false
```